src/bp-groups/bp-groups-functions.php

```diff
@@ -3545,6 +3545,7 @@ function bp_get_active_group_types( $args = array() ) {
 	$args = bp_parse_args( $args, array(
 		'posts_per_page' => - 1,
 		'post_type'      => bp_groups_get_group_type_post_type(),
+                'post_status'    => 'publish',
```

Comment:
@dkvasani Make the tabbing proper use the PHPCBF to make this proper.

Reply:
made tabbing proper